plugins/reporters/static-html/src/main/kotlin/StaticHtmlReporter.kt

```diff
@@ -174,7 +174,7 @@ class StaticHtmlReporter : Reporter {
         return document.serialize().normalizeLineBreaks()
     }
 
-    private fun getRuleViolationSummaryString(ruleViolations: List<ResolvableViolation>): String {
+    private fun getRuleViolationSummaryString(ruleViolations: List<TableReportViolation>): String {
```

Comment:
Should this also use plural "Table*s*Report..." to not have too many variants of the spelling, and to document that it belongs to the scope of `TablesReport`?

Reply:
Or maybe just move these classes inside of `TablesReport` and omit the prefix, similar to like you did for the `Row` classes?